docs/KEYS-AND-CERTS.md

```diff
@@ -21,16 +21,10 @@ If EVE is running on hardware with a trusted platform module (TPM), then these k
 | Onboarding key | Used initially for onboarding | ECC (P-256) | /config/onboard.key.pem | [Registration](REGISTRATION.md) |
 | Onboarding cert | Used initially for onboarding | ECC (P-256) | /config/onboard.cert.pem |  [Registration](REGISTRATION.md) |
 | Endorsement cert | Needed for some use of vTPM | Depends on TPM | Generated by TPM | [Identity of EVE](SECURITY.md#identity-of-eves-instance) |
+| Vault Encryption | Lock/unlock vault |  | Sealed to TPM PCRs  | [Encrypted Data Store](SECURITY.md#encrypted-data-store) |
+| Vault Encryption | Backup for upgrades |  | Encrypted and sent to controller  | [Encrypted Data Store](SECURITY.md#encrypted-data-store) |
 
-### Storage keys
-
-EVE implements a vault for application content and volumes in /persist/vault/. This is encrypted using standard filesystem encryption (fscrypt for the ext4 file system and native ZFS encryption for the ZFS file system).
-
-But master key for this is managed using EVE using the TPM to seal the keys under the TPM PCRs, and with an encrypted copy of the master key being sent to the controller as a backup, and which the controller will share with the device after successful attestation as specified in [Measured Boot and Remote Attestation](https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation).
```

Comment:
Do we have this reference to the measured boot document somewhere else in the docs directory?

Reply:
I see you added it further down.